### PR TITLE
proj: add stader labs org + repos to the hedera config 

### PR DESCRIPTION
**Pull request by prabhakar087**

### CODE REVIEW

File: data/ecosystems/h/hedera.toml [hunk 1]
```diff
@@ -2578,6 +2579,9 @@ url = "https://github.com/srikrishnakaashyap/CertiChain"
 [[repo]]
 url = "https://github.com/stader-labs/hbarx-cli"
 
+[repo]]
```
Comment (eherrerosj) [inline data/ecosystems/h/hedera.toml:2582]: plz fix this typo

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a comment

File: data/ecosystems/h/hedera.toml (hunk 1 above)
Comment (prabhakar087) [reply data/ecosystems/h/hedera.toml]: Fixed, thanks!

**Review by eherrerosj — APPROVED** [review]
thank you!

**Pull request merged**